native/packages/react-native-bpk-component-badge/stories.js

```diff
@@ -64,13 +67,22 @@ const style = StyleSheet.create({
   inverse: {
     backgroundColor: colorGray700,
   },
+  iconDestructive: {
+    color: colorWhite,
+  },
+  iconOutline: {
+    color: colorWhite,
+  },
```

Comment:
These styles will be unnecessary once we change it to automatically colour the accessory view.